java-rx-smgrsdk/src/main/java/io/vantiq/utils/SvcConnSvcMsgCodec.java

```diff
@@ -0,0 +1,59 @@
+package io.vantiq.utils;
```

Comment:
We're putting SvcConnSvcMsg instances on the bus. This requires a codec even though everything is guaranteed to be local to a single server. This is all boilerplate